GPU/GPUCommon.cpp

```diff
@@ -633,11 +633,16 @@ void GPUCommon::CheckConfigChanged() {
 	if (configChanged_) {
 		gstate_c.useFlags = CheckGPUFeatures();
 		drawEngineCommon_->NotifyConfigChanged();
-		shaderManager_->DirtyLastShader();  // Don't think this is needed, at all.
 		textureCache_->NotifyConfigChanged();
+		framebufferManager_->NotifyConfigChanged();
 		BuildReportingInfo();
 		configChanged_ = false;
 	}
+
+	// Check needed when running tests.
```

Comment:
Well, it's really that the software renderer has no, and has no need for a, framebuffer manager.  It's not just tests.  Probably several of these things crash now when using the software renderer, since it wasn't in common before.

-[Unknown]

Reply:
Oh, it's fine because NotifyConfigChanged() does nothing on softgpu.

-[Unknown]